crates/aptos/src/move_tool/bytecode.rs

```diff
@@ -429,8 +467,32 @@ impl BytecodeCommand {
             .map_err(|err| CliError::UnexpectedError(format!("Unable to disassemble: {}", err)))
     }
 
-    fn disassemble_v2(&self, bytecode_path: &Path) -> Result<String, CliError> {
+    fn disassemble_v2(
+        &self,
+        bytecode_path: &Path,
+        print_code_size: Option<Vec<String>>,
+    ) -> Result<String, CliError> {
         let bytecode_bytes = read_from_file(bytecode_path)?;
+        let print_code_bool = print_code_size.is_some_and(|v| {
+            (v.len() == 1 && v[0].is_empty())
+                || v.contains(
+                    &bytecode_path
+                        .file_name()
+                        .unwrap_or_default()
+                        .to_string_lossy()
+                        .to_string(),
+                )
+        });
+        if print_code_bool {
+            println!(
+                "Code size of module {} is: {} kbs",
+                bytecode_path
+                    .file_name()
+                    .unwrap_or_default()
+                    .to_string_lossy(),
+                bytecode_bytes.len() / 1024
```

Comment:
Just a general remark: of this is a vector of Bytecode instructions? If so, our bytecode is 32 bytes (lol), so we should keep that in mind :)